core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionIntegrationSuite.scala

```diff
@@ -49,7 +49,7 @@ class BlockManagerDecommissionIntegrationSuite extends SparkFunSuite with LocalS
         .set(config.DECOMMISSION_ENABLED, true)
         .set(config.STORAGE_DECOMMISSION_ENABLED, isEnabled)
       sc = new SparkContext(conf)
-      TestUtils.waitUntilExecutorsUp(sc, 2, 6000)
+      TestUtils.waitUntilExecutorsUp(sc, 2, 60000)
```

Comment:
Oops..I wondered why it's flaky at this place before and I thought it was already 60s....

Reply:
ah, I guess it's a typo : )